app/platform_api/swagger.json

```diff
@@ -11348,7 +11363,7 @@
           "Payments"
         ],
         "summary": "Retrieve payment PDF",
-        "description": "Retrieves a PDF file for this payment if it has been synced using an app enrollment to one of the supported apps.\r\n\r\nQuickbooks Online supports AR Payments.\n\n### Roles\n\nTo call this endpoint, you must have one of these roles:\n\n* Group Owner\n* Group Admin\n* Member\n* Read-Only\n\n\nYou can view your roles with the [Status](https://developer.lockstep.io/reference/get_api-v1-status) API.",
+        "description": "Retrieves a PDF file for this payment if it has been synced using an app enrollment to one of the supported apps.\r\n\r\nQuickBooks Online supports AR Payments.\n\n### Roles\n\nTo call this endpoint, you must have one of these roles:\n\n* Group Owner\n* Group Admin\n* Member\n* Read-Only\n\n\nYou can view your roles with the [Status](https://developer.lockstep.io/reference/get_api-v1-status) API.",
```

Comment:
i see a new schema being added after we generate the schema from Swagger of platform , shouldn't we update them on ReadME also ?

Reply:
Readme is our documentation , they dont need to be updated.